sw/device/rom_exts/rom_ext_common.ld

```diff
@@ -43,7 +22,14 @@ ENTRY(_rom_ext_start_boot)
  * s19->slm conversion scripts are not able to handle non-word aligned sections.
  */
 SECTIONS {
-  .rom_ext_manifest ORIGIN(eflash) : {
+  /**
+   * Phony section to create an offset for different slots.
+   */
+  .phony ORIGIN(eflash) : {
+    . = . + _slot_offset;
+  } > eflash
```

Comment:
I should remove the `phony section`, and instead align `.rom_ext_manifest` to `ALIGN(_slot_offset)`. It is cleaner, and also eliminates possible surprises with the page size alignment, for reference see `-z max-page-size` option in `man ld`, and [LLVMN discussion](https://reviews.llvm.org/D56205). It should not be an issue if we explicitly set the alignment of every section.

Reply:
You can choose a specific address to start a section at and the linker should honour it.

Reply:
This goes away with the [comment](https://github.com/lowRISC/opentitan/pull/3949#discussion_r514255215).

Also, `ALIGN` would not have worked as that would increase slot B size. The `phony` section works, however the solution in the above comment is neater.